apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html

```diff
@@ -701,85 +701,80 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                         </div>
                     </ng-container>
 
                     <div class="row">
                         <div class="col-xs-12">
                             <textarea rows="3" cols="" [(ngModel)]="vm.selectedLedger.description" class="form-control"
-                                placeholder="Description"></textarea>
+                                      placeholder="Description"></textarea>
                         </div>
                     </div>
 
-                    <ng-template>
-                        <div class="clearfix adjust-receipt font-14">
-                            <p class="bold">Adjusted Receipts <a href="javascript:void(0)"><i
-                                        class="icon-edit11 font-10"></i></a></p>
-                            <p class="font-12 text-light mb-1">₹2400.00</p>
-                            <div class="clearfix adjustment-field">
-                                <div class="row">
-                                    <div class="col-md-4 col-sm-5 pr-05">
-                                        <div class="form-group">
-                                            <label>Adjustment Voucher<span class="text-danger">*</span></label>
-                                            <input type="text" class="form-control mb-1" value="INV2019374-3" disabled>
-                                            <input type="text" class="form-control mb-1" value="INV2019374-3" disabled>
-                                        </div>
+                    <div class="clearfix adjust-receipt font-14" *ngIf="isAdjustedInvoicesWithAdvanceReceipt">
+                        <p class="bold">Adjusted
+                            <ng-container *ngIf="vm.selectedLedger.voucherGeneratedType ==='sales'">Receipts</ng-container>
+                            <ng-container *ngIf="vm.selectedLedger.voucherGeneratedType==='receipt'">Amount</ng-container> <a href="javascript:void(0)" *ngIf="!selectedAdvanceReceiptAdjustInvoiceEditMode" (click)="openAdjustInvoiceEditMode();$event.stopPropagation();"><i
+                                class="icon-edit11 font-10"></i></a></p>
+                        <p class="font-12 text-light mb-1">{{ vm.baseCurrencyDetails?.symbol}}{{vm.selectedLedger?.invoiceAdvanceReceiptAdjustment?.totalAdjustmentAmount | giddhCurrency}}</p>
+                        <div class="clearfix adjustment-field">
+                            <div class="row" *ngFor="let item of vm.selectedLedger?.invoiceAdvanceReceiptAdjustment?.adjustedInvoices; let entryIdx = index;">
+                                <div class="col-md-4 col-sm-5 pr-05">
+                                    <div class="form-group">
+                                        <label>Adjustment <ng-container *ngIf="vm.selectedLedger.voucherGeneratedType ==='sales'" >Voucher</ng-container> <ng-container *ngIf="vm.selectedLedger.voucherGeneratedType ==='receipt'" >Invoices</ng-container> <span class="text-danger">*</span></label>
```

Comment:
Can be rewritten as:
```suggestion
                                        <label>Adjustment>(vm.selectedLedger.voucherGeneratedType === 'sales') ? Voucher : Invoices<span class="text-danger">*</span></label>
```

Reply:
No we can not use this because there may be different voucherType like cash creditNote debitNote, contra etc